build/squire-raw.js

```diff
@@ -2093,6 +2093,7 @@ var findPreviousTextOrNotEditable = function(root, node){
 
 Squire.prototype.enter = function (self, event, range) {
   self  = self  ? self  : this
+  event.preventDefault()
```

Comment:
And if the build produces a diff it prevents the push and sends 5 volts of electricity through the keyboard